src/main/groovy/de/triplet/gradle/play/PlayPublishTask.groovy

```diff
@@ -47,4 +49,14 @@ class PlayPublishTask extends DefaultTask {
         }
     }
 
+    def getTrackByName(String trackName) {
```

Comment:
This can be written in a more Groovy-ish way:

```
    def getTrackByName(String trackName) {
        def tracksList = edits.tracks().list(variant.applicationId, editId).execute()
        return tracksList.getTracks().find { it.getTrack() == trackName }
    }
```